static/lib/resources.md

```diff
@@ -172,6 +172,12 @@
 
     Real-time chat application built using hapi.js, Socket.io and Redis Pub/Sub with end-to-end tests.
 
+*   [Building a modern backend with TypeScript, Hapi, PostgreSQL and Prisma](https://www.prisma.io/blog/series/modern-backend-bdes2ps5kibb)
+
+    A series of live streams and articles on building a backend with TypeScript, Hapi, PostgreSQL, and Prisma.
+    The series covers data modeling, CRUD, aggregations, REST, validation with Joi, testing, authentication, authorization, integration with other APIs, continuous     integration and deployment with GitHub Actions and Heroku.
+    The series explores and demonstrates different patterns, problems, and architectures for a modern backend by solving a concrete problem: a grading system for       online courses. This is a good example because it features diverse relations types and is complex enough to represent a real-world use case.
+
```

Comment:
```suggestion
    A series of live streams and articles on building a backend with TypeScript, Hapi, PostgreSQL, and Prisma.
    The series covers data modeling, CRUD, aggregations, REST, validation with Joi, testing, authentication, authorization, integration with other APIs, continuous integration and deployment with GitHub Actions and Heroku.
    The series explores and demonstrates different patterns, problems, and architectures for a modern backend by solving a concrete problem: a grading system for online courses. This is a good example because it features diverse relations types and is complex enough to represent a real-world use case.

```

Reply:
I cleaned up some multiple spaces 🤷‍♂️

Reply:
See before
<img width="128" alt="Screen Shot 2020-09-24 at 18 28 03" src="https://user-images.githubusercontent.com/1328733/94173114-b5163300-fe93-11ea-8ee4-b5a709b96c03.png">